crates/astria-composer/src/executor/bundle_factory/snapshots/astria_composer__executor__bundle_factory__tests__sized_bundle_tests__snapshots.snap

```diff
@@ -0,0 +1,8 @@
+---
+source: crates/astria-composer/src/executor/bundle_factory/tests.rs
+expression: bundle.rollup_counts
+---
+{
+  "0000000000000000000000000000000000000000000000000000000000000000": 1,
+  "0101010101010101010101010101010101010101010101010101010101010101": 1
```

Comment:
could you make these slightly more complex? Add a count of 2 for example, make the rollup IDs non-zero